cornucopia/src/load_schema.rs

```diff
@@ -0,0 +1,91 @@
+use miette::NamedSource;
+use postgres::Client;
+
+use crate::utils::db_err;
+
+use self::error::Error;
+
+/// Load schema into a database. Take a list of paths as arguments and load them in order.
+/// If path is a file we load its content without checking its name.
+/// If path is a directory we load its child SQL files.
+pub fn load_schema(client: &mut Client, paths: Vec<String>) -> Result<(), Error> {
+    for path in paths {
+        let metadata = std::fs::metadata(&path).map_err(|err| Error::Io {
+            path: path.clone(),
+            err,
+        })?;
+        // List files path to load
+        let files = if metadata.is_file() {
+            vec![path]
```

Comment:
Same question as before, this will load single files of *any* extension (not just `.sql`). Is that really what we want?

Reply:
Yes, this provide a way to force execution on a file without having to change names

Reply:
Oh I see "If path is a file we load its content without checking its name". That could be surprising to users, and I don't see the benefit of not checking the filename for the sake of avoiding typos, but maybe you have a usecase for it?

Reply:
Either way, I won't block the PR on this, I'm just curious smile

Reply:
You are right, we should check the name until someone complain

Reply:
> Yes, this provide a way to force execution on a file without having to change names

To me, the disparity between the single file and directory behavior for this is surprising. I just want to be sure we're leaving the least amount of surprises for end users.

Reply:
Should we error ?

Reply:
And for directory do we ignore non .sql files or we error on them ?